frontend/src/pages/TermPlanner/DraggableCourse/DraggableCourse.tsx

```diff
@@ -117,32 +120,43 @@ const DraggableCourse = ({ code, index, term }: Props) => {
                     style={{ fontSize: '16px', color: theme.warningOutlined.color }}
                   />
                 ))}
-              <S.CourseLabel>
-                {isSmall ? (
-                  <Text className="text">{code}</Text>
-                ) : (
-                  <div>
-                    <Text className="text">
-                      <strong>{code}: </strong>
-                      {title}
-                    </Text>
-                  </div>
-                )}
-                {showMarks && (
-                  <div>
-                    <Text strong className="text">
-                      Mark:{' '}
-                    </Text>
-                    <Text className="text">
-                      {/*
-                      Marks can be strings (i.e. HD, CR) or a number (i.e. 90, 85).
-                      Mark can be 0.
-                    */}
-                      {typeof mark === 'string' || typeof mark === 'number' ? mark : 'N/A'}
-                    </Text>
-                  </div>
-                )}
-              </S.CourseLabel>
+              <Badge
```

Comment:
can we override it via styled components? Also, I notice some weird issue with the `line-height`. Could you fix this (the issue with the line height comes from the css in `Badge` since we are wrapping it within the course label. Either override the `line-height` css property of do an absolute position with the badge (similar to the uoc badge)